src/test/java/com/orzechazo/eshop/controllers/OrderControllerE2ETest.java

```diff
@@ -0,0 +1,137 @@
+package com.orzechazo.eshop.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.orzechazo.eshop.bootstrap.tests.BootstrapUsersAndOrders;
+import com.orzechazo.eshop.domain.Order;
+import com.orzechazo.eshop.domain.dto.OrderDto;
+import com.orzechazo.eshop.exceptions.ResourceNotFoundException;
+import com.orzechazo.eshop.repositories.OrderRepository;
+import com.orzechazo.eshop.repositories.UserRepository;
+import com.orzechazo.eshop.services.OrderServiceImpl;
+import com.orzechazo.eshop.services.UserServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.math.BigDecimal;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Objects;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ExtendWith(SpringExtension.class)
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class OrderControllerE2ETest {
+    @Autowired
+    OrderRepository orderRepository;
+    @Autowired
+    UserRepository userRepository;
+    OrderServiceImpl orderService;
+    UserServiceImpl userService;
+    OrderController orderController;
+    MockMvc mockMvc;
+    private static final String DB_ORDER_ID = BootstrapUsersAndOrders.DB_ORDER_ID1;
```

Comment:
Don't assign one constant to another one, whole purpose of static constant is to use them across many classes, just use here ones defined in Bootstrap class